packages/server/utils/analytics/analytics.ts

```diff
@@ -445,8 +448,12 @@ class Analytics {
     this.track(userId, 'Reset Groups Clicked', {meetingId, teamId})
   }
 
-  private track = (userId: string, event: AnalyticsEvent, properties?: Record<string, any>) =>
-    this.segmentAnalytics.track(userId, event, properties)
+  private track = (userId: string, event: AnalyticsEvent, properties?: Record<string, any>) => {
+    Promise.all([
```

Comment:
+1 this is redundant because we're not `await`ing anything, i.e. just calling
```ts
      this.amplitudeAnalytics.track(userId, event, properties)
      this.segmentAnalytics.track(userId, event, properties)
```
without `Promise.all` would do the same.